classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  attr_accessor :label
```

Comment:
- [ OPTIONAL ] Hey there! 👋 It's widely embraced as a best community practice to place your getters and setters at the top of your class definition. 🚀 This helps in providing a clear and organized structure to your class, making it easier for developers to quickly understand and access the properties of the class. 📚💻 Keep your code clean and readable! ✨

Reply:
oh, thanks @zeff96 , I will make these changes to follow best practices.